ComponentKitTests/CKComponentViewAttributeTests.mm

```diff
@@ -81,15 +129,15 @@ - (void)testThatMountingViewWithPrimitiveAttributeActuallyAppliesAttributeToView
   CKComponent *testComponent = [CKComponent newWithView:{[CKNSNumberView class], {
     {@selector(setSelected:), YES},
     {CKComponentViewAttribute::LayerAttribute(@selector(setOpacity:)), @0.5},
-    {@selector(setTag:), 2},
+    {@selector(setTag:), (NSInteger)2},
```

Comment:
Not really happy about me having to do this. Sending an `int` to a `long` setter does work as intended (due to the typecast we do to the `IMP`). But checking type encodings in debug mode fails the assert here.